lib/fizzy/execute.cpp

```diff
@@ -446,7 +446,7 @@ T fnearest(T value) noexcept
 }
 
 template <typename T>
-__attribute__((no_sanitize("float-divide-by-zero"))) inline constexpr T fdiv(T a, T b) noexcept
+[[gnu::no_sanitize("float-divide-by-zero")]] inline constexpr T fdiv(T a, T b) noexcept
```

Comment:
@chfast is it useful merging these changes?

Reply:
Yes.

Reply:
Apparently it doesn't work as the sanitizer task reports division-by-zero.